sys/net/link_layer/ieee802154/submac.c

```diff
@@ -374,14 +374,15 @@ int ieee802154_submac_init(ieee802154_submac_t *submac, const network_uint16_t *
     ieee802154_phy_conf_t conf =
     { .phy_mode = submac->phy_mode,
       .channel = CONFIG_IEEE802154_DEFAULT_CHANNEL,
-      .page = CONFIG_IEEE802154_DEFAULT_CHANNEL,
+      .page = CONFIG_IEEE802154_DEFAULT_SUBGHZ_PAGE,
       .pow = CONFIG_IEEE802154_DEFAULT_TXPOWER };
```

Comment:
Currently a different channel page is saved to `submac->channel_page` (and `channel_num` too) when 2.4 GHz or Sub-1-GHz is supported, maybe we can use these values there?

Reply:
yes, of course. I completely miss that one